### PR TITLE
Fixing #158 and #204

### PR DESCRIPTION
**Pull request by tyler-ball**
#158 
#204 

I actually didn't fix these bugs, I just thought I did.

Apparently 

\cc @irvingpop @dblessing @jkeiser

### CODE REVIEW

File: lib/chef/provisioning/aws_driver/version.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Chef
 module Provisioning
 module AWSDriver
-  VERSION = '1.3.0.dev.0'
+  VERSION = '1.2.1'
```
Comment (randomcamel) [inline lib/chef/provisioning/aws_driver/version.rb]: Intentional?

Comment (tyler-ball) [inline lib/chef/provisioning/aws_driver/version.rb]: Yeah - I plan to quickly release 1.2.1 after merging this to master, and then change it back to `1.2.0.dev.1` post-release.

**Comment by dblessing** [comment]
Awesome. Thanks @tyler-ball

**Comment by irvingpop** [comment]
:+1:  this resolves the "leaky instances" issue for me

**Comment by pburkholder** [comment]
Before:

```
[2015-05-28T10:53:18-04:00] ERROR: aws_launch_configuration[peterb-sensu-client] (@recipe_files::/Users/pburkholder/Projects/pburkholder/conjur_demo/cookbooks/sensu_client/recipes/aws.rb line 94) had an error: NoMethodError: undefined method `aws_tags' for Chef::Resource::AwsLaunchConfiguration
```

After:

:dancers:

**Pull request merged**